corpus/metadata/location_specifier.csv

```diff
@@ -3925,6 +3930,8 @@ Q6178293,Mariestad
 Q6178293,Stockholm
 Q6178299,Göteborg
 Q6178341,Gammelstad
+Q6178389,"Eksjö, Axel P"
```

Comment:
This seem to be a bug to fix? Maybe in wikidata?

Reply:
The iort on wikidata also included first name. I edited that, so this should disappear next time we query from wikidata.